app/scripts/services/session.js

```diff
@@ -7,13 +7,32 @@
  * @namespace hmdaPilotApp
  * @module {Service} Session
  */
-module.exports = /*@ngInject*/ function () {
+module.exports = /*@ngInject*/ function ($cookies, $cookieStore) {
 
-    var session = {
-        verifiedQualityEdits: [],
-        verifiedMacroEdits: {},
-        verifiedSpecialEdits: {},
-        verifiedIRSReport: false
+    var validPassword = 'p1l0t',
+        session = {
+            verifiedQualityEdits: [],
+            verifiedMacroEdits: {},
+            verifiedSpecialEdits: {},
+            verifiedIRSReport: false
+        },
+        loginDialogOptions = {
```

Comment:
We keep the dialog config for the progress bar in a config file. Should this config (and the password) go there too?